mesonbuild/cargo/interpreter.py

```diff
@@ -668,6 +662,8 @@ def _create_lib(self, pkg: PackageState, build: builder.Builder,
             build.identifier(_extra_args_varname()),
             build.identifier('system_deps_args'),
             build.identifier('env_args'),
+            build.identifier('lint_args'),
+            build.string(f'--edition={pkg.manifest.package.edition}'),
         ]
```

Comment:
Use rust_std here, otherwise looks good.

Reply:
Oh you mean as override_option?